app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,7 @@
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_pack_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```

Comment:
I think this line might be duplicated. What do you think?

Reply:
I think that this is not duplicated , the first link is for the stylesheets in app/assets/stylesheets and the last one is for webpacker stylesheet

Reply:
@marianoalarcon

Reply:
I think the same as Ange, in fact we follow the following guide in case it is useful for someone:
[](https://medium.com/daily-web-dev/use-bootstrap-4-with-ruby-on-rails-6-and-webpack-fe7300604267)